src/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -24,6 +24,12 @@ public static void MinValue()
         }
 
         [Fact]
+        public static void UnixEpoch()
```

Comment:
You need to skip the added tests on the full framework test run by adding the attribute

```
        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The full .NET framework not supporting UnixEpoch yet")]
```

Reply:
Looks like that didn't skip netfx. Investigating.

Reply:
I don't think `[SkipOnTargetFramework]` is going to work since these new APIs don't yet exist in .NET Framework. Instead, you need to create new DateTimeTests.netcoreapp.cs and DateTimeOffsetTests.netcoreapp.cs files for these tests (see other similar files in the directory), and conditionally include them in [System.Runtime.Tests.csproj](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System.Runtime.Tests.csproj) like the other *.netcoreapp.cs files.

Reply:
That makes sense. Tests updated, build is running. Thank you!